base/process.jl

```diff
@@ -548,68 +548,75 @@ spawn_opts_inherit(stdios::StdIOSet) = (stdios,)
 spawn_opts_inherit(in::Redirectable=RawFD(0), out::Redirectable=RawFD(1), err::Redirectable=RawFD(2), args...) =
     ((in, out, err), args...)
 
-"""
-    spawn(command)
-
-Run a command object asynchronously, returning the resulting `Process` object.
-"""
-spawn(cmds::AbstractCmd, args...; chain::Union{ProcessChain, Nothing}=nothing) =
-    spawn(cmds, spawn_opts_swallow(args...)...; chain=chain)
+_spawn(cmds::AbstractCmd, args...; chain::Union{ProcessChain, Nothing}=nothing) =
+    _spawn(cmds, spawn_opts_swallow(args...)...; chain=chain)
 
 function eachline(cmd::AbstractCmd; chomp=nothing, keep::Bool=false)
     if chomp !== nothing
         keep = !chomp
         depwarn("The `chomp=$chomp` argument to `eachline` is deprecated in favor of `keep=$keep`.", :eachline)
     end
     stdout = Pipe()
-    processes = spawn(cmd, (DevNull,stdout,STDERR))
+    processes = _spawn(cmd, (DevNull,stdout,STDERR))
```

Comment:
Are we now unable to open a process without reaching into the internal API?

Reply:
I don't think so. Is there a case not covered by `open` and `pipeline`?